src/actions/analytics.ts

```diff
@@ -85,22 +78,7 @@ export const trackScreenChange = (
     'mh',
   );
 
-  const sendScreenChange = (MCID: string) => {
-    const context: TrackStateContext = {
```

Comment:
I removed this because the firebase screen function only accepts the screen name and no additional parameters. This is weird though as the native versions do allow the passing of addtional parameters.